README.md

```diff
@@ -22,65 +22,51 @@ Each of these two main folders contain domain folders. Each domain folder is
 named to the integration `domain` and must match the domain set in the
 integration `manifest.json` file.
 
-A domain folder can contain four files:
+A domain folder can contain the following files:
 
 - `icon.png`: A square avatar-like icon, representing the brand or product for that domain.
 - `logo.png`: The logo of the brand or product for that domain.
 - `icon@2x.png`: hDPI version of `icon.png`
 - `logo@2x.png`: hDPI version of `logo.png`
 
-Those images are served in the following format:
+Each of those images may also have a dark theme variant for a total of 8 files. They are served in the URL format:
 
-- `https://brands.home-assistant.io/[domain]/icon.png`
-- `https://brands.home-assistant.io/[domain]/logo.png`
-- `https://brands.home-assistant.io/[domain]/icon@2x.png`
-- `https://brands.home-assistant.io/[domain]/logo@2x.png`
-- `https://brands.home-assistant.io/_/[domain]/icon.png`
-- `https://brands.home-assistant.io/_/[domain]/logo.png`
-- `https://brands.home-assistant.io/_/[domain]/icon@2x.png`
-- `https://brands.home-assistant.io/_/[domain]/logo@2x.png`
+```
+https://brands.home-assistant.io/[domain]/{dark_}[icon|logo]{@2x}.png
```

Comment:
Please revert this. This might work for a programmer, but lots of users don't understand this at all.

This is also completely unrelated to the change provided.

Reply:
The relation was to be consistent when showing a URL with the search parameter below, without repeating a list of 8 nearly identical lines.  Could just do something like `.../[domain]/[image]` instead?